pkg/nvml/return.go

```diff
@@ -19,17 +19,27 @@ import (
 )
 
 // nvml.ErrorString()
-func ErrorString(r Return) string {
-	if err := GetLibrary().Lookup("nvmlErrorString"); err != nil {
-		return fallbackErrorStringFunc(r)
-	}
-	return nvmlErrorString(r)
+func (l *library) ErrorString(r Return) string {
+	return r.Error()
```

Comment:
We either need to keep the lookup logic as before or reset the default if the library is unloaded.

Reply:
I can't leave it as before because in the r.Error() and r.String() methods I don't have access to `l`. I will update it reset the function on unload.

Reply:
Updated.